website/api/zkvm/composition.md

```diff
@@ -0,0 +1,43 @@
+# Proof Composition
+
+RISC Zero's zkVM is the only zkVM that offers a convenient API for efficiently verifying proofs inside the guest program.
+
+This feature enables rich possibilities for building modular applications. Check out the [blog post] to learn more about the power of proof composition.
+
+## Start Building
+
+To use proof composition in your program, you'll need to call [`add_assumption()`] on the host-side and [`env::verify()`] on the guest-side, as demonstrated in this [code example].
```

Comment:
The high level idea of assumptions and resolution could definitely use an introduction first. It might useful to introduce the hypothetical idea of actually just running the verifier code in the zkVM, which is what we tried first, and then explain how the assumptions mechanism improves upon that. That's kind of what I tried to do in my talk for the study club.

Reply:
https://www.youtube.com/watch?v=x0-7Y46bQO0 I just watched an needed to re-watch the assumptions bit 5 times to get it to sink in :sweat_smile: . I had the "naive" model of executing the verifier in another guest, the recursion stuff was new to me with no other exposure to the concept prior